pep-0703.rst

```diff
@@ -1531,6 +1531,52 @@ author is not aware of a way to add write barriers to CPython without
 substantially breaking the C-API.
 
 
+Why Not Deprecate ``PyDict_GetItem`` in Favor of ``PyDict_FetchItem``?
+----------------------------------------------------------------------
+
+This PEP proposes a new API ``PyDict_FetchItem`` which behaves like
+``PyDict_GetItem``, but returns a new reference instead of a borrowed
```

Comment:
It's totally optional and at your discretion as the PEP author, but just FYI, you can cross-reference Python and C functions, classes/types, methods/members, etc., and they will be automatically linked to the Python docs. So, for example, you could use `` :c:func:`PyDict_GetItem` `` here to make this render as [`PyDict_GetItem`](https://docs.python.org/3/c-api/dict.html?highlight=pydict_getitem#c.PyDict_GetItem), if you wanted.